src/helpers/irma_button.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import p from 'prop-types';
+
+import './irma_button.scss';
+
+class IrmaButton extends React.Component {
```

Comment:
Those are called [function components](https://reactjs.org/docs/components-and-props.html). They are used when it is a simple component, mostly without any state. Then you'll skip all the boilerplate of the class definitions and constructors and immediately start specifying the render. That improves readability. That makes sense for all helpers (at least that's how all simple widget components are called now; that name can of course change).

I saw that we use function components for some more complex components in the `components` directly too. I'll check whether we can re-write those. There it is maybe a bit weird indeed.

Reply:
I converted `LogsTable` to a full React component, because that one was very ugly. And the other ones don't seem that problematic to me now. We can maybe convert them when we add all the `propTypes`, because then the function components pattern becomes a bit more ugly.